simplq/src/components/common/Button/Button.stories.jsx

```diff
@@ -37,3 +37,7 @@ export const RefreshButton = (props) => {
     </StandardButton>
   );
 };
+
+export const ForkOnGithubButton = () => {
+  return <StandardButton>Fork on Github</StandardButton>;
+};
```

Comment:
The link to fork should be added here too, right?

Reply:
yes, let's just on click send the user to our repo home https://github.com/SimplQ/simplQ-frontend